system/netdata-openrc.in

```diff
@@ -0,0 +1,83 @@
+#!/sbin/runscript
+
+# The user netdata is configured to run as.
+# If you edit its configuration file to set a different
+# user, set it here too, to have its files switch ownership
+: ${NETDATA_OWNER:=netdata:netdata}
```

Comment:
we deal with this file at #166, I guess better to leave it as-is to avoid conflicts for now.